app/models/manageiq/providers/amazon/manager_mixin.rb

```diff
@@ -68,10 +68,12 @@ def verify_credentials(args)
       region           = args["provider_region"]
       default_endpoint = args.dig("authentications", "default")
 
-      access_key, secret_access_key, proxy_uri, service_account = default_endpoint&.values_at(
-        "userid", "password", "proxy_uri", "service_account"
+      access_key, secret_access_key, service_account = default_endpoint&.values_at(
+        "userid", "password", "service_account"
       )
 
+      proxy_uri = http_proxy_uri.to_s
```

Comment:
If the proxy is _not_ defined, will an empty string being passed into raw_connect cause a problem?

Reply:
Yeah I just noticed this and it actually works but is confusing, fixing